README.md

```diff
@@ -31,6 +31,11 @@ Aerobridge is fully open source and you can get involved by participating in our
 
 This is an open source implementation of the "Manufacturer's Management Server" to help with key signing and managing interactions with India's [Digital Sky](https://digitalsky.dgca.gov.in/) API infrastructure. This server can be deployed to any public and private cloud and be used to manage communication with DGCA's Digital Sky Infrastructure and manage NPNT permissions. For a more technical introduction, see this [presentation](https://docs.google.com/presentation/d/1cZrNwNrLtLIj5eKEGql2HN-G1gZFbbGhGbiTB1i16So/edit?usp=sharing).
 
+## Dependencies
```

Comment:
Recommend adding a text to explain what this is about / for. e.g. If you want to have a snapshot these libraries are necessary. Also perhaps a sentence saying that this is a Django app.

Reply:
Added a description as comment about why those packages are required